internal/collector/nginxossreceiver/README.md

```diff
@@ -0,0 +1,44 @@
+# NGINX Receiver
+
+This receiver can fetch stats from a NGINX instance by using two different ways of scraping:  
```

Comment:
Perhaps reword this to "This receiver can fetch metrics from a NGINX instance via two sources:"